Singer.API/ClientApp/src/app/modules/admin/components/singerevents/singerevent-overview/singerevent-overview.component.ts

```diff
@@ -82,6 +82,11 @@ export class SingerEventOverviewComponent implements OnInit, AfterViewInit {
       });
    }
 
+   // Returns true if the max number of registrants for the event have been exceeded
+   getMaxRegistrantsExceeded(row: SingerEvent):boolean {
+      return row.currentRegistrants > row.maxRegistrants ? true : false;
```

Comment:
This can be simplified to:
`return row.currentRegistrants > row.maxRegistrants;` because the return value of that statement will be `true` or `false`

Reply:
I would call this method `isMaxRegistrantsExceeded`